src/stories/packages/components/Icon.stories.ts

```diff
@@ -0,0 +1,35 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { allIconIds, Icon } from '@terra-ui-packages/components';
+
+const meta: Meta<typeof Icon> = {
+  title: 'Packages/Components/Icon',
+  component: Icon,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    icon: {
+      control: 'select',
+      options: allIconIds,
+      description: 'ID of icon to be used from IconLibrary',
+    },
+    size: {
+      control: 'number',
+      description: 'icon size',
+      table: {
+        defaultValue: { summary: '16' },
+      },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Icon>;
+
+export const IconUsage: Story = {
```

Comment:
I wasn't sure what to name this so I named it `IconUsage`. Happy to change it!

Reply:
We settled on the default story name being Example.